lua/nvim-treesitter/ts_utils.lua

```diff
@@ -234,7 +233,7 @@ function M.find_definition(node, bufnr)
   for _, def in ipairs(locals.get_definitions(bufnr)) do
     for _, def_node in ipairs(M.get_local_nodes(def)) do
       if M.get_node_text(def_node)[1] == node_text then
-        table.insert(matching_def_nodes, def_node)       
```

Comment:
@steelsojka please use a linter! Your refactoring introduced a lot of post-spaces. We should probably add a linter job as a CI test.